internal/sirius/users.go

```diff
@@ -0,0 +1,101 @@
+package sirius
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+type UserStatus string
+
+func (us UserStatus) String() string {
+	return string(us)
+}
+
+func (us UserStatus) TagColour() string {
```

Comment:
You could start the component library off with a `govuk-tag.gotmpl` file, instead of having the class names in code

Reply:
I'm not too keen on this, since we'd either need to do context-specific string matching in `govuk-tag.gotmpl`; or pass through colours and trust there's a class matching them.

At least with the existing code we can search for `govuk-tag--` and check them.